.github/workflows/code.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         timeout-minutes: 10
 
       - name: Build binaries
-        run: cargo build --release
+        run: cargo build --release --features try-runtime
```

Comment:
Oh, this is actually a problem. We need to run E2E tests for the regular builds...

Reply:
Redesigned